### DISCUSSION TITLE
Slack alerts

### DISCUSSION THREAD

**Discussion by jjkirn** [Security-Onion-Solutions/securityonion#3096] [Unsupported Versions]
I read the info from the 2.3.2 section (https://docs.securityonion.net/en/2.3/) regarding setting up slack alerts and am a bit confused. First thing is when I search for slack alerts it sends me to "ElastAlert". When I go to that section it says to go to /opt/so/conf/elastalert/ . 
At that location there is a file elastalert_config.yaml.  The docs say to add something like:
------------------
alert:
- "slack":
    slack_webhook_url: "https://hooks.slack.com/services/YOUR_WEBHOOK_URI"
------------------
I tried that - no alerts send to my slack (yes I used my WEBHOOK_URI) and after a reboot it looks like that file gets overwritten anyways.
Second, on the Security Onion Web console there is a section  "Alerts" - alerts are coming in from suricata and ossec - not ElasticSearch (ElastaAlert). I am looking to get these alerts sent to Slack - I'm not sure if I have to go through setting up ElastAlert to get these alerts.

So my question is - How do I set up Security Onion 2.3.X to send the alerts that show up on the "Alerts" section of the Security Onion Web console to my Slack web hook URI?

**Comment by cm-ops** [comment]
You would use Elastalert to do this. Your rule is placed on the Manager in `/opt/so/rules/elastalert`.

You would add your Slack information to the alert portion of your rule you set up.

**Reply by jjkirn** [reply]
Could you point me to an example of a rule and does it need to be in a specific file or do you just add it to the bottom of the elastalert_config.yaml file?

**Reply by cm-ops** [reply]
Something similar to this https://github.com/Security-Onion-Solutions/securityonion/discussions/2141. Instead of the email alert portion, you would put the Slack information.
